### ISSUE TITLE
Flow - Ensure images provide informative alternative text - (2036569902)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5571]
### Module: 
 13 Flow 

### Violation: 
 Ensure images provide informative alternative text 

### WCAG Reference: 
- [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html) 

### Instance ID: 
 2036569902 

### Severity: 
 10 

### Description: 
 [Issue]
There is decorative image "Nature" with textual equivalent. 

[User Impact]
Screen reader users will receive unnecessary and potentially confusing information.

[Code Reference] 
 ```
<img src="https://placeimg.com/300/200/nature" alt="nature">
 ``` 

### Note: 
 [Suggestion]
Ensure images provide informative alternative text. Non-meaningful images must be marked as decorative, which causes screen readers to ignore them. To mark an ``` <img> ``` element as decorative, set its alt attribute to "" (that is, alt="" exactly as written, with no space between the quotation marks). 

[Compliant Code Example] 
 ```
<img src="https://placeimg.com/300/200/nature" alt="">
 ```
 

### Media Type: 
 Images 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Not a valid issue, related to the testing page. Closing.
